pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -4705,8 +4705,6 @@ SELECT cur_max_builtin_oid FROM [SELECT max(oid) as cur_max_builtin_oid FROM pg_
 ## existing functions at init time. Though the changes to builtin function OID is
 ## generally ok, it's still better if we could move the new implement to end of the
 ## list at init time (see all_builtins.go)
-## TODO(chengxiong): consider to have a deterministic list of builtin function oids
```

Comment:
The fixed oid thing for builtin functions (#84509) was not backported, so like to keep this test here even though adding a builtin through a backport in the future should be super rare..